mrp_project/__manifest__.py

```diff
@@ -7,7 +7,7 @@
     """,
     "author": "Solvos",
     "license": "LGPL-3",
-    "version": "15.0.1.1.0",
+    "version": "15.0.2.1.0",
```

Comment:
When a version is updated, numbers at its right should be reset (e.g. from `1.3.0` to `2.0.0` or from `1.2.1` to `1.3.0`):
```suggestion
    "version": "15.0.2.0.0",
```